docs/about/learning-resources.rst

```diff
@@ -47,3 +47,5 @@ Guide to Hiring and Getting Hired
 This is a growing set of resources for those who want to hire or be hired as a documentarian.
 
 You can check out the :doc:`/hiring-guide/index` here.
+
+You can check out the :doc:`/community-spotlight/index` here.
```

Comment:
We still want to add this to the `toctree` on line 15 as well. It's how Sphinx builds out the sidebars and navigation for the site.

Reply:
Like this?

Reply:
On line 15...